docs/framework/toc.yml

```diff
@@ -31,24 +18,28 @@
     href: install/index.md
   - name: For developers
     href: install/guide-for-developers.md
-  - name: Windows 10 and Windows Server 2016
-    href: install/on-windows-10.md
-  - name: Windows 8.1 and Windows Server 2012 R2
-    href: install/on-windows-8-1.md
-  - name: Windows 8 and Windows Server 2012
-    href: install/on-windows-8.md
-  - name: Windows 7 SP1 and Windows Server 2008 R2
-    href: install/on-windows-7.md
-  - name: Windows Vista and Windows Server 2008
-    href: install/on-windows-vista.md
-  - name: Windows XP and Windows Server 2003
-    href: install/on-windows-xp.md
+  - name: By version
```

Comment:
@thraka I added subfolders "By version" and "Troubleshoot" because I think it makes this section of the TOC cleaner. Please approve or disapprove.

Reply:
Ping @thraka.

Reply:
Got approval through Teams.